src/sidebar/components/test/annotation-body-test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const { createElement } = require('preact');
+const { mount } = require('enzyme');
+
+const AnnotationBody = require('../annotation-body');
+const mockImportedComponents = require('./mock-imported-components');
+
+describe('AnnotationBody', () => {
+  function createBody(props = {}) {
+    return mount(<AnnotationBody text="test comment" {...props} />);
+  }
+
+  beforeEach(() => {
+    AnnotationBody.$imports.$mock(mockImportedComponents());
+  });
+
+  afterEach(() => {
+    AnnotationBody.$imports.$restore();
+  });
+
+  it('displays the comment if `isEditing` is false', () => {
```

Comment:
"the comment" meaning "the annotation body content?"

Reply:
Yes.